ansible/roles/nim-waku/defaults/main.yml

```diff
@@ -14,6 +14,9 @@ nim_waku_node_key_file_path: '{{ nim_waku_cont_vol }}/nodekey'
 # WARNING: Enabling this will disable relaying of messages
 nim_waku_light_node_enabled: false
 
+# Default value to Indicate if the swap Protocol will be enabled
+nim_waku_swap_protocol_enabled: true
```

Comment:
Comments shouldn't state what the line does, but why it's there.
The fact that Swap Protocol is enabled with this is self explanatory due to the name of the variable in question.
A useful comment would explain what the swap protocol is and what it provides for the user so they know if they want the option enabled or not.